cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,25 @@
+import socket
+import time
+import os
+
+from django.core.management import BaseCommand
+
+
+class Command(BaseCommand):
+    help = "Waits for database to be ready for connection"  # noqa: VNE003
+
+    def handle(self, *args):
+        port = int(os.environ["POSTGRES_PORT"])  # 5432
+
+        db_socket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+
+        print("Waiting for DB connection...")
```

Comment:
It's better to use Django's `self.stdout.write` method for command line output instead of `print`. It provides a more consistent interface and allows you to control verbosity levels for different kinds of output.